templates/plugin/gutterball/foreman_gutterball_url.yaml.erb

```diff
@@ -0,0 +1,6 @@
+:foreman_gutterball_url:
+  :protocol: <%= @protocol %>
```

Comment:
we could go either way. i was doing similar to what is found in katello's config

for example:
```yaml
common:
  # ...
  candlepin:
    url: https://localhost:8443/candlepin
```